lightly/utils/hipify.py

```diff
@@ -15,6 +17,18 @@ class bcolors:
 
 
 def print_as_warning(message: str, warning_class: Type[Warning] = UserWarning) -> None:
+    """Prints a warning message with custom formatting.
+
+    Temporarily overrides the default warning format to apply custom styling, then
+    restores the original formatting after the warning is printed.
+
+    Args:
+        message:
+            The warning message to print.
+        warning_class:
+            The type of warning to raise. Defaults to `UserWarning`.
```

Comment:
See also the the [instructions in the main issue](https://github.com/lightly-ai/lightly/issues/1216):

> Do not document default values if they are part of the function definition. E.g. if the function is defined as: def fun(value: int = 5) there is no need to document that 5 is the default for value in the docstring. It is already visible in the function definition.


```suggestion
            The type of warning to raise.
```